src/Combinatorics/Graphs/structs.jl

```diff
@@ -1,7 +1,13 @@
 import Oscar: Polymake
-import Oscar.Polymake: Directed, Undirected
+import Oscar.Polymake:
+  Directed, Undirected,
+  EdgeMap, NodeMap
 
-struct Graph{T <: Union{Directed, Undirected}}
+@attributes mutable struct Graph{T <: Union{Directed, Undirected}}
   pm_graph::Polymake.Graph{T}
+  
+  function Graph{T}(nverts::Int) where T  <: Union{Directed, Undirected}
+    return new{T}(Polymake.Graph{T}(nverts))
+  end
```

Comment:
```suggestion
  end
  function Graph{T}(pmg::Polymake.Graph{T}) where T <: Union{Directed, Undirected}
    return new{T}(pmg)
  end
```
If you add a custom inner constructor you also need to add a default one.